core/src/bmi.jl

```diff
@@ -633,6 +633,11 @@ function BMI.update_until(model::Model, time)::Model
     return model
 end
 
+function BMI.update_subgrid_level(model::Model)::Model
+    update_subgrid_level!(model.integrator)
+    return model
+end
```

Comment:
```suggestion
function update_subgrid_level(model::Model)::Model
    update_subgrid_level!(model.integrator)
    return model
end
```

I don't think creating a function in an external package is the right approach here, since this is not a BMI function. You also need to add a C callable version in https://github.com/Deltares/Ribasim/blob/main/build/libribasim/src/libribasim.jl to get it in libribasim.

Reply:
You're obviously right, not really awake yet I guess...